objcopy.go

```diff
@@ -108,6 +108,10 @@ func Objcopy(infile, outfile string) error {
 
 	// Write to the file, in the correct format.
 	switch filepath.Ext(outfile) {
+	case ".gba":
+		// The address is not stored in a .gba file.
+		_, err := f.Write(data)
+		return err
 	case ".bin":
```

Comment:
Why not add it here?
```go
case ".bin", ".gba":
```

Reply:
Because I suspect there will be some additional further steps for gba files in the near future, and this makes it easier for people who do not know TinyGo as well to add them.

Reply:
Because I suspect there will be some additional further steps for gba files in the near future, and this makes it easier for people who do not know TinyGo as well to add them.

Reply:
Ah, that's a good point.